tests/lib/prepare.sh

```diff
@@ -931,6 +990,11 @@ setup_reflash_magic() {
         core_name="core20"
     elif os.query is-core22; then
         core_name="core22"
+    elif os.query is-core24; then
+        core_name="core24"
+        # TODO: revert this once snaps are ready in target channel
+        KERNEL_CHANNEL=edge
```

Comment:
We should never use edge for the kernel. Only beta. I do not think we need to override it here. `beta` will work fine.